src/app/modules/shared/components/dialog/dialog.component.html

```diff
@@ -0,0 +1,17 @@
+<div class="modal-dialog" role="document">
+  <div class="modal-content" *ngIf="list">
+    <div class="modal-header">
+      <h5 class="modal-title" id="exampleModalLabel">{{ title }}</h5>
+      <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+        <span aria-hidden="true">&times;</span>
+      </button>
+    </div>
+    <div class="modal-body">
+      Are you sure you want to delete <b>{{ list.project_name || list.name }}</b>
+    </div>
+    <div class="modal-footer">
+      <button (click)="removeListById(list.id)" type="button" class="btn btn-primary">Delete</button>
```

Comment:
Probably not, since this component is intended to be used just to confirm delete actions. In this sense, I'd rather call it
_src/app/modules/shared/components/dialogs/remove.dialog.component.html_

The functionality works good but we need to move the buttons so the first option is **Cancel** and the second one is **Delete**. This is how confirmation modal panels usually look like:
<img width="234" alt="Screen Shot 2020-06-16 at 09 51 06" src="https://user-images.githubusercontent.com/1638636/84790437-f9ea1e00-afb6-11ea-801f-14aac4c41257.png">

Taking the image into account, let's use the style **danger** for the **Delete** button.